### ISSUE TITLE
RangeError: Maximum call stack size exceeded after Meteor update to 1.2.0.1

### ISSUE THREAD

**Issue by marktomm** [msavin/Mongol#112]
Hello!

I have been having some problems with Mongol (1.1.5) after updating Meteor to 1.2.0.1.

When using Ctrl+m the minimized Mongol window pops up without any trouble, showing all my current collections with the number of elements in them.

When I click on a particular collection with elements, the window with the results is empty with a message 'No document found'. However the Duplicate, Update, Remove, Left and Right are present.
I also receive an error message in the console after the click:

`<Exception from Tracker recompute function:
debug.js:41 RangeError: Maximum call stack size exceeded
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:13)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)>`

If I click Right or Left Arrows in a colelction with elements, console outputs the following errors:

`<debug.js:41 Exception from Tracker recompute function:
debug.js:41 TypeError: Cannot read property 'setMembers' of undefined
    at doMaterialize (:3000/packages/meteor:/💻app/packages/blaze/view.js:359)
    at Object.Tracker.nonreactive (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:589)
    at [object Object].doRender (:3000/packages/meteor:/💻app/packages/blaze/view.js:355)
    at :3000/packages/meteor:/💻app/packages/blaze/view.js:199
    at Function.Template._withTemplateInstanceFunc (:3000/packages/meteor:/💻app/packages/blaze/template.js:457)
    at :3000/packages/meteor:/💻app/packages/blaze/view.js:197
    at Object.Blaze._withCurrentView (:3000/packages/meteor:/💻app/packages/blaze/view.js:538)
    at viewAutorun (:3000/packages/meteor:/💻app/packages/blaze/view.js:196)
    at [object Object].Tracker.Computation._compute (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:323)
    at [object Object].Tracker.Computation._recompute (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:342)
debug.js:41 Exception from Tracker recompute function:
debug.js:41 TypeError: Cannot read property 'setMembers' of undefined
    at doMaterialize (:3000/packages/meteor:/💻app/packages/blaze/view.js:359)
    at Object.Tracker.nonreactive (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:589)
    at [object Object].doRender (:3000/packages/meteor:/💻app/packages/blaze/view.js:355)
    at :3000/packages/meteor:/💻app/packages/blaze/view.js:199
    at Function.Template._withTemplateInstanceFunc (:3000/packages/meteor:/💻app/packages/blaze/template.js:457)
    at :3000/packages/meteor:/💻app/packages/blaze/view.js:197
    at Object.Blaze._withCurrentView (:3000/packages/meteor:/💻app/packages/blaze/view.js:538)
    at viewAutorun (:3000/packages/meteor:/💻app/packages/blaze/view.js:196)
    at [object Object].Tracker.Computation._compute (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:323)
    at [object Object].Tracker.Computation._recompute (:3000/packages/meteor:/💻app/packages/tracker/tracker.js:342)
debug.js:41 Exception from Tracker recompute function:
debug.js:41 RangeError: Maximum call stack size exceeded
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:13)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
debug.js:41 Exception from Tracker recompute function:
debug.js:41 RangeError: Maximum call stack size exceeded
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:13)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)
    at Object.EditableJSON.collection (:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14)>`

**Comment by Jank1310** [comment]
Same here, i get:

```
Exception from Tracker recompute function: undefined
RangeError: Maximum call stack size exceeded
at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:13:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1)
    at Object.EditableJSON.collection (http://localhost:3000/packages/meteor:/💻app/packages/babrahams_editable-json/packages/babrahams_editable-json.js:14:1) undefined
```

Update:
Meteor: 1.2.0.1
Mongol: 1.1.5

**Comment by Anima-t3d** [comment]
Same here, makes debugging a pain...

**Comment by Anima-t3d** [comment]
Fixed after meteor update to 1.2.0.2 and remove the meteortoys:allthings and add it again.

**Comment by marktomm** [comment]
Problem solved after update to Meteor 1.2.0.2 and mongol 1.2.0.
